babel.config.js

```diff
@@ -1,12 +1,5 @@
 module.exports = {
   presets: [
-    [
-      '@babel/preset-env',
```

Comment:
Consider adding this since it is important when we want to target all browser. This block helps to target IE, Opera and Safari browsers.

Reply:
Resolved in the browser